<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PyramidDialog</class>
 <widget class="QDialog" name="PyramidDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Octaves:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>OctaveLayers:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Sigma:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QSpinBox" name="octavesBox">
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="octaveLayersBox">
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="sigmaBox">
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.600000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QSpinBox" name="showPyramidLayersBox">
     <property name="value">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="showPyramidButton">
     <property name="text">
      <string>Show pyramid</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="cancleButton">
     <property name="text">
      <string>&amp;Cancle</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="computeButton">
     <property name="text">
      <string>Compute</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>octavesBox</tabstop>
  <tabstop>octaveLayersBox</tabstop>
  <tabstop>sigmaBox</tabstop>
  <tabstop>computeButton</tabstop>
  <tabstop>showPyramidLayersBox</tabstop>
  <tabstop>showPyramidButton</tabstop>
  <tabstop>cancleButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancleButton</sender>
   <signal>clicked()</signal>
   <receiver>PyramidDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
